APublicizer.cs

```diff
@@ -209,15 +209,8 @@ private void Processor<T>(Collection<T> values, Predicate<T> filter, Action<T> p
 
         private void DoFixupEvents(PublicizeResult value)
         {
-            bool FilterEvent(EventDefinition e)
-            {
-                // It'll cause a compilation error if I don't do
-                e.DeclaringType.Fields.Single(f => f.Name == e.Name).IsPublic = false;
-                // Don't lie to users
-                value.Fields--;
```

Comment:
Don't remove this code, make it execute when there's a delegate field. It's required because a public event & a public field with the same name is causing compilation errors to me if I try to access the event.

Reply:
I can't think of why this would happen but I guess it doesn't hurt to account for it.